src/Cowsay.zig

```diff
@@ -11,23 +11,17 @@ const default_cow =
     \\    ||     ||
 ;
 
-var general_purpose_allocator = std.heap.GeneralPurposeAllocator(.{}){};
-
-
-
 /// writer for output. must be an AnyWriter. Use `.any()` to convert to AnyWriter before passing in.
-w: std.io.AnyWriter,
+writer: std.io.AnyWriter,
 /// the eyes. will substitute the first two `o` of the cow
 eyes: [2]u8 = [2]u8{ 'o', 'o' },
 
-/// allocator to use when formatting message
-allocator: std.mem.Allocator = general_purpose_allocator.allocator(),
 /// private variables
 thinking: bool = false,
-max_line_length: u32 = 0,
-offset: u32 = 0,
+max_line_length: usize = 0,
```

Comment:
Using a `usize` when working with spans of memory is typically preferable, unless you know specifically what the maximum expected value is (then you can constrain just to that value, e.g. `u10` for 1023).